src/helpers/optionalStacking/common.ts

```diff
@@ -28,24 +37,111 @@ export function getViewByTwoAttributes(
  *      => ignore computeRatio and use stackMeasuresToPercent
  *
  * stackMeasures is applied only when there are [2 measures and more]
- * stackMeasuresToPercent is applied only when there are [1 measure + 1 stackBy] or [2 measures and up]
+ * stackMeasuresToPercent is applied only when there are [1 measure + 1 stackBy] or [2 measures and more]
  */
 export function getSanitizedStackingConfig(
+    buckets: VisualizationObject.IBucket[],
+    config: IChartConfig = {},
+): ISanitizedStackingConfig {
+    const clonedBuckets = cloneDeep(buckets);
+    const measuresItems = findBucketByLocalIdentifier(clonedBuckets, MEASURES).items || [];
```

Comment:
are we ok, that it works only for "measures" bucket and not "secondary..." and so on?

Reply:
`getSanitizedBucketsAndStackingConfig` supports `AreaChart`, `BarChart` and `ColumnChart` only. There are no `secondaryMeasure` and `secondaryMeasures`.

For `secondary_yaxis`, it uses the same `MEASURES` localIdentifier.

In my opinion, it's ok

Reply:
Hi @ptquang86,
In combo chart, there are 2 measure buckets (measures & secondary_measures) and should be handled in same way.
1. Ignore `computeRatio` when each bucket has multiple measures.
2. Ignore `computeRatio` for both buckets if dual axis is disabled and total of measures > 1

However, Im waiting for the confirmation from Ondrej about that (https://jira.intgdc.com/browse/SD-429)
And I think it is would be fine to handle in other PR.